src/OpenTK/Math/Vector2.cs

```diff
@@ -68,23 +68,27 @@ public Vector2(float x, float y)
         /// <summary>
         /// Gets or sets the value at the index of the Vector.
         /// </summary>
-        public float this[int index] {
-            get{
-                if(index == 0)
+        public float this[int index]
```

Comment:
Yeah... I opted to include this one because visual studio kept automatically applying, but I'll rebase and remove it for now.